google-cloud-testing/storage-benchwrapper/README.md

```diff
@@ -0,0 +1,63 @@
+# storage-benchwrapper
+
+storage-benchwraper is a gRPC wrapper around the storage library for benchmarking purposes.
+
+## Running
+
+```
+mvn clean install -DskipTests=true (one time)
+cd google-cloud-testing/storage-benchwrapper
+mvn clean install exec:java -DskipTests=true
+```
+
+## Generating .proto sources
+
+Sources are generated automatically as part of the protobuf-maven-plugin plugin,
+but if you'd like to generate them yourself to see the output you can run:
```

Comment:
Another option is if you run `mvn compile` should should be able to see the generated sources in `target/generated-sources/`

Reply:
Done